build.gradle

```diff
@@ -20,6 +20,8 @@ plugins {
     id 'com.github.sherter.google-java-format' version '0.8'
     id 'com.github.hierynomus.license' version '0.15.0'
     id 'io.codearte.nexus-staging' version '0.21.2'
+    id "com.github.ben-manes.versions" version "0.27.0"
```

Comment:
What is the reason for adding these plugins?

Reply:
didn't mean to add that - will remove it - that was used to identify newer versions

Reply:
done